### PR TITLE
Includes dynamic proxy_pass definition to make NGINX refresh the DNS lookup

### PR DESCRIPTION
**Pull request by pedrovivaldi**
### What Changed

- Includes dynamic proxy_pass definition to make NGINX refresh the DNS lookup
- Reference: https://forum.nginx.org/read.php?2,215830,215832#msg-215832

### How To Test or Verify

- Describe any steps that may help reviewers verify changes.
- Anything beyond basic unit testing, such as assistive tech usage, or special interactions.

### PR Checklist

Below are some checklists to follow for the correct procedure in different circumstance. The first list ("All PRs Checklist") should be followed for ALL PRs. The next 2 are additive to this list depending on what type of PR you are using.

For example: If you are submitting a content change to one of the support documents, your checklist would include the:

- "All PRs Checklist"
- AND the "Content Changes Checklist

If you are submitting a feature addition, enhancement, or bug fix, your checklist would include the:

- "All PRs Checklist"
- AND the "Development Changes Checklist"

#### All PRs Checklist

- [x] Give your pull request a meaningful name.
- [x] Use lowercase filenames.
- [x] Apply at least one team label according to which team is the content expert (ie. `team-FE` or `team-SAZ`)
- [ ] Pull request approval from the FE team or content experts (see label applied above) that isn't the content creator.

#### Content Changes Checklist

- [ ] Check that your article looks correct in the preview here or in a [Netlify deploy preview](https://app.netlify.com/sites/support-docs/deploys).
- [ ] Check the links in your article.
- [ ] Check the images in your article (if there are any)
- [ ] Check to make sure you are using markdown appropriately as outlined in `examples/article.md` in the root of the project directory and on the momentum doc's [preface article](https://support.sparkpost.com/momentum/4/4-preface)
- [ ] Check to make sure the [Copy and Tone Guidelines are followed](https://docs.google.com/document/d/1dej9J7N9M8lcbJXnT9kxyNB_EFBPHIBLZBbHaxRWqhQ/edit).

#### Development Changes Checklist (some checks are automatic github actions and will not be listed here. ie. "all tests pass")

- [ ] The appropriate tests are created in `cypress/` directory in the root of the project
- [ ] The lighthouse score is passing according to the [FE Support Docs' Service Outline](https://sparkpost.atlassian.net/wiki/spaces/ENG/pages/1238728726/FE+Support+Docs) SLI/SLOs

### CODE REVIEW

File: content/docs/tech-resources/using-proxy-https-tracking-domain.md [hunk 1]
```diff
@@ -54,6 +54,9 @@ sudo apt-get install nginx
 
 On a Debian distribution, this command will install nginx with a sample configuration, located at **/etc/nginx/**.  To enable a reverse proxy back to SparkPost for your tracking domain, see the sample configuration file below (sample tracking domain is click.nddurant.com).
 
+Note: we need to set the `proxy_pass` directive with a variable so the ip address of the domain will be looked up
+dynamically and result will be cached for 5 minutes. Otherwise, it would be only looked up at server startup
```
Comment (mtgray-sp) [inline content/docs/tech-resources/using-proxy-https-tracking-domain.md:58]: Can you please reword this to:
> Note: you must store spgo.io in a variable so that nginx re-resolves the domain when its TTL expires.

I believe that you also need to include the `resolver` directive when using this method [1]. Please check me on that.

[1] https://www.nginx.com/blog/dns-service-discovery-nginx-plus/

File: content/docs/tech-resources/using-proxy-https-tracking-domain.md [hunk 2]
```diff
@@ -54,15 +54,20 @@ sudo apt-get install nginx
 
 On a Debian distribution, this command will install nginx with a sample configuration, located at **/etc/nginx/**.  To enable a reverse proxy back to SparkPost for your tracking domain, see the sample configuration file below (sample tracking domain is click.nddurant.com).
 
+Note: you must store `spgo.io` in a variable so that nginx re-resolves the domain when its TTL expires. You also have to include the `resolver` directive to explicitly specify a DNS server to resolve the hostname. By including the `valid` parameter to the directive, you can tell nginx to ignore the TTL and to re‑resolve names at a specified frequency. In the sample below, nginx re‑resolves names every 10 seconds.
+
 ```apacheconf
+resolver 10.0.0.2 valid=10s
```
Comment (mtgray-sp) [inline content/docs/tech-resources/using-proxy-https-tracking-domain.md:60]: We're missing a semicolon here

File: content/docs/tech-resources/using-proxy-https-tracking-domain.md (hunk 1 above)
Comment (mtgray-sp) [reply content/docs/tech-resources/using-proxy-https-tracking-domain.md]: There's another NGINX config snippet later in this document that we need to update, too.

**Review by llacava — APPROVED** [review]
